packages/flex-plugin-scripts/src/scripts/__tests__/deploy.test.ts

```diff
@@ -123,7 +123,7 @@ describe('deploy', () => {
 
     it('should throw an exception if custom is called without an argument', async (done) => {
```

Comment:
should the description change too

Reply:
Na, it's still a "custom" version. Everything bumps version.